cypress/integration/examples/mainTest.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("main test", () => {
+  it("open main site and check console", () => {
+    cy.visit("http://localhost:8080")
+    
+  });
+});
+
+Cypress.on('window:before:load', (win) => {
```

Comment:
Could you add it to beforeAll?

Reply:
do you mean something like this? 

`beforeEach(() => {
  Cypress.on('window:before:load', (win) => {
    cy.spy(win.console, 'error');
    cy.spy(win.console, 'warn');
  });
})
`

Reply:
yes, just beforEach will be executed before every it test.
Don't we want just to attach it once?